src/Mod/Sketcher/App/SketchObject.cpp

```diff
@@ -2635,6 +3002,17 @@ int SketchObject::fillet(int GeoId1, int GeoId2, const Base::Vector3d& refPnt1,
                             oc2pf.x,
                             oc2pf.y,
                             oc2pf.z);
+
+        /*auto printoffsetcurve = [](Part::GeomOffsetCurve *c) {
+
+            for(double param = c->getFirstParameter(); param < c->getLastParameter(); param = param
+        + (c->getLastParameter()-c->getFirstParameter())/10) Base::Console().Log("\n%f:
+        (%f,%f,0)\n", param, c->pointAtParameter(param).x,c->pointAtParameter(param).y);
+
+        };
+
+        printoffsetcurve(ocurve1);
+        printoffsetcurve(ocurve2);*/

```

Comment:
IDK.  I think either the whole set of #ifdef DEBUG are irrelevant, or it should all be there.  Because it's fillet I left it.  Will add comment as this shows a useful technique.